playground/workflows/__tests__/workflows.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { expect, test, vi } from 'vitest';
+import { getJsonResponse } from '../../__test-utils__';
+
+test('creates a Workflow with an ID', async () => {
+	const instanceId = 'test-id';
+
+	expect(await getJsonResponse(`/create?id=${instanceId}`)).toEqual({
+		id: instanceId,
+		status: {
+			status: 'running',
+			output: [],
+		},
+	});
+
+	await vi.waitFor(
+		async () => {
+			expect(await getJsonResponse(`/get?id=${instanceId}`)).toEqual({
+				status: 'running',
+				output: [{ output: 'First step result' }],
+			});
+		},
+		{ timeout: 5000 },
```

Comment:
Here and below, can you explain why such a long timeout is needed?

Reply:
I based this off some tests in `workers-sdk` so used the timeouts from there. We can shorten it if you like.

Reply:
I see, I was just wondering, I thought there might have been a specific reason for this (e.g. you saw it failing in CI or something)

I'm fine with it 👍